contracts/communityFee/OPFCommunityFeeCollector.sol

```diff
@@ -28,7 +28,6 @@ contract OPFCommunityFeeCollector is Ownable {
         address payable newCollector,
```

Comment:
Why does this contract _not_ inherit from an interface? (There may be a good reason! Just to me right now it looks inconsistent.)

Reply:
Thanks. This is a pragmatic dividing line. OK then.